src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerUtils.java

```diff
@@ -65,6 +65,8 @@
     // get realigned incorrectly.  See https://github.com/broadinstitute/gatk/issues/5060
     public static final int MINIMUM_READ_LENGTH_AFTER_TRIMMING = 10;
 
+    // this notation can be interpreted as a representation of the alleles present on the two phased haplotypes at the site:
+    // "0": REF or '*'; "1": site-specific alt allele
```

Comment:
Let's capitalize these constants: `PHASE_01`.

Reply:
Done, and converted to enums as per your comment below.